renovate.json

```diff
@@ -4433,6 +4433,23 @@
       "minimumReleaseAge": "7 days",
       "enabled": true
     },
+    {
+      "groupName": "Kibana ES|QL Highlighting",
+      "matchDepNames": [
+        "@elastic/monaco-esql"
+      ],
+      "reviewers": [
+        "team:kibana-esql"
+      ],
+      "matchBaseBranches": [
+        "main"
+      ],
+      "labels": [
+        "Team:ESQL",
+        "release_note:skip"
```

Comment:
__question:__ what about [`backport:*` label](https://docs.google.com/document/d/1VyN5k91e5OVumlc0Gb9RPa3h1ewuPE705nRtioPiTvY/edit?tab=t.0)?

Reply:
Added it here: https://github.com/elastic/kibana/pull/219394/commits/e10b450a1e61f52992d3bd107367ae9d1226f753

Reply:
Personally I find the the all-open and prev-major labels very confusing Oleg 🙈 With the backport:version and the version labels you know exactly where you are backporting. I find it less confusing and I think many people from the analyst-xp agree with me

Reply:
> But how does the backport:version label work without version labels?

But we are adding the version labels of course

Reply:
>  This is why I think it might make sense to not have a backport label here in order the PR to complain (?)

My main concern here would be that it would be so easy to add `backport:skip` to silence it. If `backport:version` complained until you added versions, that would probably be the best trade-off (I'm not sure how it works today, maybe it already works like that).

Anyway, let's see how it goes and if `backport:skip` becomes a wide-spread problem then we can re-group 👍

Reply:
yes this makes sense